test/e2e/upgrades/ingress.go

```diff
@@ -141,7 +144,7 @@ func (t *IngressUpgradeTest) Teardown(f *framework.Framework) {
 	framework.CleanupGCEIngressController(t.gceController)
 }
 
-func (t *IngressUpgradeTest) verify(f *framework.Framework, done <-chan struct{}, testDuringDisruption bool) {
+func (t *IngressUpgradeTest) verify(f *framework.Framework, done <-chan struct{}, testDuringDisruption, pollForGCPResources bool) {
```

Comment:
can we have 
```
type ingressUpgradeVerifyOptions int

const (
  testDuringDisruption ingressUpgradeVerifyOptions = 1 << iota
  pollForGCPResources ingressUpgradeVerifyOptions = 1 << iota
)

func ... verify(f, done, ingressUpgradeVerifyOptions)
```

Reply:
Thanks for the suggestion, but these verify options might not fit into this switch option model. `testDuringDisruption` and `pollForGCPResources` are not necessarily mutually exclusive, just that we don't have a test case that requires them to happen at the same time yet.